app/config/set_parameters.php

```diff
@@ -50,11 +50,14 @@
     }
 }
 
-foreach ($parameters['parameters'] as $key => $value) {
-    if (defined('_PS_IN_TEST_') && $key === 'database_name') {
-        $value = 'test_'.$value;
+if ($container instanceof \Symfony\Component\DependencyInjection\Container) {
```

Comment:
xdebugging my tests I've met a weird side effect of our test suite: in some cases, it's the Ioc of PrestaShip which was called here, producing a notice. This is also why I've renamed the variable in another file.